Assets/XML/Text/DynamicNames/DynamicNames_Poland.xml

```diff
@@ -38,6 +38,10 @@
 		<English>Polish-Lithuanian Commonwealth</English>
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_CIV_POLAND_EMPIRE</Tag>
```

Comment:
This has already been fixed, and is probably the reason why the branch is incompatible. Please check for compatibility first.